src/python_testing/TC_MOD_2_3.py

```diff
@@ -0,0 +1,260 @@
+            TestStep("4b", "TH sends a _ViewScene_ command to DUT with the _GroupID_ field set to _G~1~_ and the _SceneID_ field set to 0x01. DUT sends a _ViewSceneResponse_ command to TH with the _Status_ field set to 0x00 (SUCCESS), the _GroupID_ field set to _G~1~_, the _SceneID_ field set to 0x01, the _TransitionTime_ field set to 0 and a set of extension fields appropriate to the values set in step 4a."),
+            TestStep(
+                "5a", "TH sends a _AddScene_ command to DUT with the _GroupID_ field set to _G~1~_, the _SceneID_ field set to 0x02 and the ExtensionFieldSets set to: '[{ ClusterID: 0x0050, AttributeValueList: [{ AttributeID: 0x0003, ValueUnsigned8: _M~1~_ }] }]' where _M~1~_ is a value selected from the _L~1~_ but different from _M~Initial~_ obtained in step 3 unless _L~1~_ contains only one item. DUT sends a _AddSceneResponse_ command to TH with the _Status_ field set to 0x00 (SUCCESS), the _GroupID_ field set to _G~1~_ and the _SceneID_ field set to 0x02."),
+            TestStep("5b", "TH sends a _ViewScene_ command to DUT with the _GroupID_ field set to _G~1~_ and the _SceneID_ field set to 0x02. DUT sends a _ViewSceneResponse_ command to TH with the _Status_ field set to 0x00 (SUCCESS), the _GroupID_ field set to _G~1~_, the _SceneID_ field set to 0x02, the _TransitionTime_ field set to 0 and a set of extension fields where CurrentMode attribute value is equal to _M~1~_ from step 5a."),
+            TestStep("6a", "TH sends a _RecallScene_ command to DUT with the _GroupID_ field set to _G~1~_, the _SceneID_ field set to 0x02 and the _TransitionTime_ omitted."),
+            TestStep("6b", "After a few seconds, TH reads the _CurrentMode attribute_ from DUT. Value has to be the same as _M~1~_ from step 5a."),
+            TestStep("7a", "TH sends a _RecallScene_ command to DUT with the _GroupID_ field set to _G~1~_, the _SceneID_ field set to 0x01 and the _TransitionTime_ omitted."),
+            TestStep("7b", "After a few seconds, TH reads the _CurrentMode attribute_ from DUT. Value has to be the same as _M~Initial~_ from step 3."),
+
+        ]
+        return steps
+
+    @async_test_body
+    async def setup_test(self):
+        super().setup_test()
+        # Pre-Condition: Commissioning
+        self.step("0")
+
+        self.step("0a")
+
+        self.TH1 = self.default_controller
+        self.kGroupKeyset1 = 0x01a1
+        self.groupKey = Clusters.GroupKeyManagement.Structs.GroupKeySetStruct(
+            groupKeySetID=self.kGroupKeyset1,
+            groupKeySecurityPolicy=Clusters.GroupKeyManagement.Enums.GroupKeySecurityPolicyEnum.kTrustFirst,
+            epochKey0="0123456789abcdef".encode(),
+            epochStartTime0=1110000,
+            epochKey1="0123456789abcdef".encode(),
```

Comment:
Epoch keys  don't match what's on the test plan:
```suggestion
            epochKey1="b0b1b2b3b4b5b6b7b8b9babbbcbdbebf".encode(),
```

Reply:
I believe this was intentional due to the fact that I believe the default accepted keys on CI are those. putting the epoch keys as in the test plan resulted in a CI failure, alternatively we could also put a check to use the current keys when doing a CI run and the test plan keys during cert events.

Reply:
Oh I see, I didn't know there were CI specific epoch keys.

If that's the case, are those defined anywhere we could load from instead of hardcoding in every test? How do other tests do this?